CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 **Bug Fixes**:
 
 - Use correct commandline argument name for setting Relay port. ([#1059](https://github.com/getsentry/relay/pull/1059))
+- Retrieve OS Context for Unity Events INGEST-293. ([#1072](https://github.com/getsentry/relay/pull/1072))
```

Comment:
_nit_: Since this is new functionality, should we rather put this under "Features"?

Reply:
Well arguably we are fixing broken functionality, we are not recognising/parsing correctly the os info string, but as you wish, in my opinion it is a bug fix.

Reply:
ok, makes sense. let's keep it in bugfix then